docs/presentation/Content/PropertyCategorySpecification.md

```diff
@@ -1,56 +1,117 @@
-# PropertyCategory Content Modifier
+# Property Category Specification
 
 > TypeScript type: [PropertyCategorySpecification]($presentation-common).
 
 This content modifier allows defining a custom property category. Simply defining it doesn't affect the content,
-but a defined category can be used in property overrides.
+but a defined category can be used in [property overrides](./PropertySpecification.md).
```

Comment:
```suggestion
Content modifier for defining custom property categories. Custom categories start out empty, but can be populated using [property overrides](./PropertySpecification.md).
```

Reply:
I don't think the suggestion is correct. Unreferenced categories don't start out empty - they're not even created.